src/tests/ftest/pool/rebuild_no_cap.yaml

```diff
@@ -2,29 +2,31 @@
 # required quantity is indicated by the placeholders
 hosts:
   test_machines:
-    - boro-A
-    - boro-B
-    - boro-C
-    - boro-D
-    - boro-E
-    - boro-F
+    test_servers:
+      - boro-A
+      - boro-B
+      - boro-C
+      - boro-D
+      - boro-E
+      - boro-F
+    test_clients:
```

Comment:
Same as before. If it's just single client process, we can leave that for avocado's default client agent.

Reply:
Based upon the similar conversation in #703, can this be resolved?